pydicom/dataset.py

```diff
@@ -961,14 +961,7 @@ def _get_PIL_supported_compressed_pixeldata(self):
                     try:
                         decompressed_image = PILImg.open(fio)
                     except IOError as e:
-                        try:
```

Comment:
Use `e.strerror` instead of `e.message`, that is an attribute of `IOError` both in Python 2 and 3 (same below).